### ISSUE TITLE
Sprint jumping on straight paths

### ISSUE THREAD

**Issue by Karang** [PrismarineJS/mineflayer-pathfinder#86]
To increase speed, the bot should have an option to sprint-jump on straight path. Needs careful tuning for overshoots and tights spaces.

**Comment by GenerelSchwerz** [comment]
Agreed. Manually setting the control state of the bot to sprint jump when traveling in a straight path results in the pathfinder being much less efficient.

**Comment by Project-Magenta** [comment]
e
